checker/tests/nullness/JavaCopExplosion.java

```diff
@@ -3,6 +3,7 @@
 
 @org.checkerframework.framework.qual.DefaultQualifier(Nullable.class)
 class Explosion {
+    // :: error: (extends.clause.invalid)
```

Comment:
Is this error telling us something useful? What is invalid about this declaration?
For the Nullness Checker, when do you expect this kind of error?